format_notification.py

```diff
@@ -0,0 +1,72 @@
+#!/bin/env python
+
+# this while loop is for not considering two string fields which you get right after starting a process
+# Test string ->['string "ShanonPark posted in "HEY! YOU! What are you up to right now! (Part 3)" - Linkin Park / LPU Forums"'] 2 
+
+import time
+import sys
+import notify2
+from setup import main
+
+def EditString(data: str) -> str:
+    return data[0].split('"', 1)[1].replace('"', '')
+
+def Notify(summary, data, urgency, timeout=800000):
+    notify2.init("AutoReplyToLP")
+    pop_up = notify2.Notification(summary, data)
+    pop_up.set_urgency(urgency)
+    pop_up.set_timeout(timeout)
+    return pop_up
+
+def countdown_timer(timeMS):
```

Comment:
Not sure why some are Notify and others are countdown_timer instead of CountdownTimer

I am used to having Captial letter starts for class names, then function and variables to be named like countdownTimer (Pascal vs Camel case)

Doesn't matter the convention you choose, but try to stick to one.

Reply:
okay thanks.. I'll keep lowerCamelCase from now
the only benefit i think is to read and understand the code better